src/App.jsx

```diff
@@ -43,7 +44,16 @@ function App() {
           <p>
             Contrast ratio is the different in lightness between two colors.
           </p>
-          <p>Requirements according to <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html">WCAG 2.0</a> and <a href="https://www.w3.org/WAI/WCAG21/Understanding/contrast-enhanced.html">WCAG 2.1:</a></p>
+          <p>
```

Comment:
Please don't reformat the code here.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
the prettier just keeps formatting it 👀
is there any way to ignore that line?

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
@emmalearnscode issue solved! I ignored the line from formatting using `prettier-ignore` comment!

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good